weblate/html/list-important-projects.html

```diff
@@ -0,0 +1,25 @@
+{% load i18n %}
+{% load translations %}
+
+<table class="sort table">
+<thead>
+<tr>
+<th>{% trans "Project" %}</th>
+<th colspan="2">{% trans "Translated" %}</th>
+</tr>
+<tbody>
+{% for prj in important_projects %}
+{% with prj.get_important_translated_percent as percent %}
+<tr>
+<th><a href="{{ prj.get_absolute_url }}">{{ prj }}</a></th>
+<td class="progress-cell">{% important_translation_progress prj %}</td>
+<td class="percent">{{ percent }}%</td>
+</tr>
+{% endwith %}
+{% endfor %}
+</tbody>
+</table>
+
+<p>
+{% include "legend.html" %}
+</p>
```

Comment:
This is almost same as list-projects.html, it would be better if the code can be shared...